src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/DefaultRazorLanguageServerCustomMessageTarget.cs

```diff
@@ -718,12 +719,12 @@ public override async Task<VSInternalWrapWithTagResponse> RazorWrapWithTagAsync(
                                 csharpRanges.AddRange(result);
                             }
                         }
-                    });
+                    }, cancellationToken);
 
             }
 
-            var htmlDocument = GetHtmlDocumentSnapshsot(foldingRangeParams.TextDocument.Uri.ToUri());
-            var htmlRanges = new List<OmniSharp.Extensions.LanguageServer.Protocol.Models.FoldingRange>();
+            var htmlDocument = GetHtmlDocumentSnapshsot(foldingRangeParams.TextDocument.Uri);
```

Comment:
`Uri.ToUri()` 😅

Reply:
Yeah, it was a bit silly before. If I recall `TextDocument.Uri` was actually of type `DocumentUri`. Fun.